### DISCUSSION TITLE
events not showing up in piwik (debugger)?

### DISCUSSION THREAD

**Discussion by rudgr** [endgameapp/piwikpro-gtm-serverside-ga4#2] [Q&A]
Initial setup seems to be OK and we see the request being send from our GTM SST tagging server to your data collection endpoint (https://domain.piwik.pro/ppms.php?rec=1&idsite=… ).

We also see a HTTP 202 OK Status message coming back.

However we don’t see any data appearing in piwik pro, also not in the piwik debugger.

Are we missing anything here?

tx in advance,

PS: cross-post from: [https://community.piwik.pro/t/issues-with-server-side-gtm-integration-for-ga4/1750](https://community.piwik.pro/t/issues-with-server-side-gtm-integration-for-ga4/1750)

**Comment by endgameapp** [comment]
Can you provide some more info? Like screenshots of the tag configuration, trigger etc

**Reply by rudgr** [reply]
I sent you a DM

**Reply by rudgr** [reply]
seems azure related, potential fix here: https://github.com/endgameapp/piwikpro-gtm-serverside-ga4/pull/5
